schema/models_directory/pb/qm/dft/functional/components_directory/local_functional.json

```diff
@@ -1,7 +1,7 @@
 {
     "schemaId": "models-directory/pb/qm/dft/functional/components-directory/local-functional",
     "$schema": "http://json-schema.org/draft-04/schema#",
-    "title": "local functional component schema",
+    "title": "LocalFunctional",
```

Comment:
It seems with the conversion of schema to TS interface the titles cannot be chosen freely anymore. I think we should put a comment in the readme to remember the role of `title` when adding a new schema.

Reply:
Added test that check schema titles